etc/greenboot.d/motd/redboot.motd

```diff
@@ -0,0 +1 @@
+Health Check FAILED! Boot Status is RED!
```

Comment:
should we tell the user how to see which ones failed?

Reply:
good idea! Thinking about it, this message could also be generated dynamically to directly show what failed, similar to what Cockpit does for displaying its ip/url. will add a note for using the journal for now, and open an issue for a more dynamic generation of it.